Explorer/Assets/DCL/ExplorePanel/SystemMenuController.cs

```diff
@@ -111,9 +115,13 @@ private void Logout()
             async UniTaskVoid LogoutAsync(CancellationToken ct)
             {
                 Web3Address address = web3IdentityCache.Identity!.Address;
+
                 await web3Authenticator.LogoutAsync(ct);
+
                 profileCache.Remove(address);
-                await userInAppInitializationFlow.ExecuteAsync(true, true, world, playerEntity, ct);
+
+                await unloadAllScenes.ExecuteAsync(ct);
```

Comment:
Should i move this call inside `userInAppInitializationFlow.ExecuteAsync(...)` by passing a new argument? Something like `bool destroyAllScenes`